src/SonarScanner.MSBuild.Shim/TFSProcessor.Wrapper.cs

```diff
@@ -0,0 +1,107 @@
+﻿/*
+ * SonarScanner for MSBuild
+ * Copyright (C) 2016-2019 SonarSource SA
```

Comment:
Minor: wrong copyright date.
Probably worth bulk updating separately from the PR.

Reply:
Yep there is one already here : https://github.com/SonarSource/sonar-scanner-msbuild/pull/895